frontend/components/Course/StudentQueuePage/QuestionForm.tsx

```diff
@@ -51,15 +54,48 @@ const QuestionForm = (props: QuestionFormProps) => {
         }
     };
 
-    // const getDropdownOptions = (tags) => {
-    //     return tags.map((tag) => {
-    //         return {
-    //             key: tag,
-    //             value: tag,
-    //             text: tag,
-    //         };
-    //     });
-    // };
+    useEffect(() => {
```

Comment:
Shouldn't we be able to share the tags data with EditQuestionModal? Perhaps but pulling it up to the QuestionCard component. 

Boilerplate can also be saved with resource hook as described below.